src/test/javascript/spec/component/learning-paths/graph/learning-path-graph.component.spec.ts

```diff
@@ -58,4 +83,20 @@ describe('LearningPathGraphComponent', () => {
         expect(centerStub).toHaveBeenCalledOnce();
         expect(centerStub).toHaveBeenCalledWith(true);
     });
+
+    it('should change view mode and load data if necessary', () => {
+        comp.viewMode = LearningPathViewMode.GRAPH;
+        comp.learningPathId = 1;
+        fixture.detectChanges();
+        expect(getLearningPathNgxGraphStub).toHaveBeenCalledOnce();
```

Comment:
Feel free to ignore but I discovered this recently. Jest has a matcher called `toHaveBeenCalledExactlyOnceWith` that allows you to combine both your assertions into one.